apps/MUXSDKStatsExampleSPM/MUXSDKStatsExampleSPM.xctestplan

```diff
@@ -35,6 +35,9 @@
       }
     },
     {
+      "skippedTests" : [
+        "MUXSDKStatsExampleSPMUITests\/testCustomSessionDataExample()"
```

Comment:
The test that'll use the local HLS server is skipped when running the non-CI app scheme.